compiler/parser/parser.cpp

```diff
@@ -426,9 +426,7 @@ static void ensureRequiredStandardModulesAreParsed() {
             if (mod == NULL) {
               INT_FATAL("Trying to rename a standard module that's part of\n"
                         "a file defining multiple\nmodules doesn't work yet;\n"
-                        "see "
-                        "test/modules/bradc/modNamedNewStringBreaks.future "
-                        "for details");
+                        "see b2397b0052 and a94694b2aa for details");
```

Comment:
I'm not sure what to do here.

b2397b0052 added the cited future.
a94694b2aa removed it, not because it was fixed, but because (IIUC) the bug was specific to handling of standard modules, and the last standard module that exposed it was changed to not expose it any more.

I don't like the change I came up with at all.  But I think it's at least no less useful to a user than what's there now.

Reply:
I guess maybe open a new github issue, and point the user at that?  I don't feel particularly qualified to open the issue, or even verify it's still present.

Reply:
I think it should simply remove any reference to the test or to the commits. I think it's somewhat likely that the bug still exists and this internal error might be useful one day, but as far as I know we don't have any standard-inculded modules that meet this pattern now.

Reply:
Leaving this to @mppf

Reply:
If someone who understands the bug could describe it in a github issue, I can point it at that.  Otherwise I'll just remove the "see ... for details" part.

Reply:
I don't think it makes sense to be a github issue at present time, please just remove the details part.